docs/UsersGuide/source/Glossary.rst

```diff
@@ -88,12 +94,18 @@ Glossary
    MPI
       MPI stands for Message Passing Interface. An MPI is a standardized communication system used in parallel programming. It establishes portable and efficient syntax for the exchange of messages and data between multiple processors that are used by a single computer program. An MPI is required for high-performance computing (HPC).
 
+   MRMS
+      Multi-Radar/Multi-Sensor (MRMS) System Analysis data. This data is required for use of the METplus verification suite within the SRW App. A two-day archive of precipitation, radar, and aviation and severe weather fields is publicly available and can be accessed `here <https://mrms.ncep.noaa.gov/data/>`__.
```

Comment:
It is required if composite reflectivity or echo top verification is desired - maybe qualify that?